Classes/Controller/ImageRenderingController.php

```diff
@@ -88,8 +88,9 @@ public function renderImageAttributes(?string $content, array $conf = []): strin
                         $processedFile = $this->getMagicImageService()
                             ->createMagicImage($systemImage, $imageConfiguration);
 
+                        $imageSource = $processedFile->getPublicUrl();
                         $additionalAttributes = [
-                            'src'    => $processedFile->getPublicUrl(),
+                            'src'    => $imageSource,
```

Comment:
I think the "fix" is in the wrong place. It should happen where the error is happening, line 186 ff: https://github.com/netresearch/t3x-rte_ckeditor_image/pull/233/files#diff-3e0e02594decda07cf6aad3ac8938fe063123b1b1ad6db2b1f3325cbbe4225fcR139

there it should check `$imageAttributes['src']` instead of `$imageSource`